### DISCUSSION TITLE
Ability to resend all queued messages in all rooms.

### DISCUSSION THREAD

**Discussion by ara4n** [element-hq/element-meta#698] [Ideas]
#### What would you like to do?

Queue up messages in multiple rooms when in bad connectivity (or offline) and then have the app send them all for me when it returns online.

#### Why would you like to do it?

Otherwise i have to remember what rooms i have unsent messages in (or go hunting for ! icons on rooms), which is very tedious and unwieldy.

#### How would you like to achieve it?

Either have the app do it automatically when connectivity returns (like WhatsApp does), or have a "send all queued msgs in all rooms" button of some kind alongside the current "send queued msgs" button.

**Comment by kittykat** [comment]
Moving to discussions as this is cross-platform :+1:
